task1/src/test/java/com/fundacionjala/movies/TestNewReleaseMovie.java

```diff
@@ -0,0 +1,37 @@
+package com.fundacionjala.movies;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by Mijhail on 14/06/2016.
+ */
+public class TestNewReleaseMovie {
+    @Test
+    public void calculateNewReleaseMovieFor1DayRental() {
+        Movie movie = new NewReleaseMovie("X-Men");
```

Comment:
If you notice you are repeating the Movie instance in all the tests
Please add a Before method
```java
private Movie movie;

@Before
public void setUp() {
    movie = new NewReleaseMovie("X-Men");
}
```